firmware/tentris/tentris.ino

```diff
@@ -234,11 +235,7 @@ void gameOver() {
 
 void fillBlock(byte x, byte y, COLOR color) {
   if (x < BOARD_WIDTH && y < BOARD_HEIGHT) {
-#ifdef TOPDOWN
```

Comment:
@WhateverFits This logic seems backwards to me? TOPDOWN is not defined, so it looks like its not using 0,0 as the bottom left of the board?

Reply:
Ok, then I probably need to tweak the snake style mapping to also be inverted. It assumed 0,0 is bottom left. Ill open another PR when I code that.